urx/onrobot_rg2_gripper.py

```diff
@@ -0,0 +1,273 @@
+
+from urx.urscript import URScript
+import time
```

Comment:
maybe it is a good idea to move all these gripper files to a directory called grippers or devices?